src/applications/disability-benefits/all-claims/tests/components/ConfirmationPoll.unit.spec.jsx

```diff
@@ -102,4 +105,114 @@ describe('ConfirmationPoll', () => {
       done();
     }, 500);
   });
+
+  describe('selectAllDisabilityNames', () => {
+    it('should return selected rated disability names', () => {
+      const state = {
+        form: {
+          data: {
+            ratedDisabilities: [
+              {
+                'view:selected': true,
+                name: 'first rated disability',
+              },
+              {
+                'view:selected': false,
+                name: 'second rated disability',
+              },
+              {
+                'view:selected': true,
+                name: 'third rated disability',
+              },
+              {
+                'view:selected': false,
+                name: 'fourth rated disability',
+              },
+            ],
+          },
+        },
+      };
+
+      const selectedDisabilities = selectAllDisabilityNames(state);
+      const { ratedDisabilities } = state.form.data;
+
+      expect(selectedDisabilities.length).to.equal(2);
```

Comment:
This is probably extraneous since you've got the deep equal below. (Same story with the next two tests.)

Reply:
Fair enough, I kept this here to help pinpoint any failures, but it probably doesn't help at all.